packages/cli/src/cmds/preview/depsChecker/dotnetChecker.ts

```diff
@@ -90,7 +90,10 @@ export class DotnetChecker implements IDepsChecker {
 
     await this._logger.debug(`[start] check dotnet version`);
     if (dotnetPath !== null && (await this.isDotnetInstalledCorrectly())) {
-      this._telemetry.sendEvent(DepsCheckerEvent.dotnetInstallCompleted);
+      // filter out global sdk
+      if (dotnetPath.includes(".fx")) {
```

Comment:
Better to check whether `dotnetPath` begins with `DotnetChecker.getDefaultInstallPath()`?